packages/common/src/api.ts

```diff
@@ -0,0 +1,15 @@
+import { IRequestOptions, getPortalUrl } from "@esri/arcgis-rest-request";
+
+/**
+ * @param requestOptions - request options that may include authentication
+ */
+export function getHubRESTAPIBaseUrl(requestOptions: IRequestOptions): string {
```

Comment:
since we already have `getPortalUrl()` maybe just `getHubBaseUrl()`?

Reply:
yes! that sounds much much better. `getHubHostUrl` was something I was wondering about as well.

Reply:
I think something w/ "API" is better than "base", maybe `getHubApiUrl()`

Reply:
`getHubUrl()`

Reply:
I think it has to have `Api` in there.

Reply:
> I think it has to have Api in there.

works for me. a code snippet for the API reference would be nice though.

```js
/**
 * ```js
 * import { getHubApiUrl() } from "@esri/hub-common";
 * //
 * getHubApiUrl({ portal: "https://custom.maps.arcgis.com/sharing/rest" })
 * >> "https://hub.arcgis.com"
 * ```
 * Retrieves the Hub API Url associated with a specific ArcGIS Online organization.
 * @param requestOptions - request options that may include authentication
 * @returns the associated Hub API Url as a string.
 */

```